doc/INSTALL

```diff
@@ -1,244 +1,215 @@
-    will set up everything properly. However you might need the following
-    paragraphs in case something goes wrong.
+  If you want to compile the KDE integration support, you obviously need KDE.
+  The kdelibs package should suffice.
 
-    Before running CMake make sure that the environment variable $QTDIR
-    points to the right location. This will help in finding the correct version
-    of Qt.
+  So before running CMake make sure that the environment variable $KDEDIR
+  points to the right location.
 
-    You might eventually set it with the command:
+  You might eventually set it with the command:
 
-    # export QTDIR="your Qt dir"
+    $ export KDEDIR="your kde dir"
 
-    On my system, Qt is installed in /usr/lib/qt4, thus I actually execute:
+  Where KDE is for e.g. installed in /usr use:
 
-    # export QTDIR=/usr/lib/qt4
+    $ export KDEDIR=/usr
 
-    To check if you are using the right version of Qt, simply run:
 
-    # qmake -v
+  ## Pthread implementation
 
-    On my system the output is:
+  This is usually included in your distribution and is probably already
+  installed. In Ubuntu or variants its libpthread-stubs0-dev.
+  The library is called libpthread.so. You can search for it using the "find"
+  or, at your option, "locate" command.
 
-    QMake version 2.01a
-    Using Qt version 4.7.2 in /usr/lib64/qt/lib
+0.  $ find / -name libpthread*
 
-    If you have multiple versions of Qt4 installed in your system and want to
-    use a specific version, exporting QTDIR won't suffice. Since the CMake
-    module searching for Qt4 uses qmake itself to determine the Qt4 library dir,
-    you can set your $PATH variable to have CMake using
-    your preferred qmake version:
+1.  $ locate -i "libpthread*"
 
-    # PATH=/opt/qt4.7/bin/:$PATH cmake ..
 
-    Please note that compiling with KDE4 support enabled will force KVIrc to
-    link against the Qt4 version that KDE4 is using. This may lead to problems
-    in linking. Disabling KDE4 support is the only known solution at the moment.
-    Note also that in this case you'll need to override the Qt libraries used
-    when running KVIrc:
+  ## GNU gettext (Optional)
 
-    # LD_LIBRARY_PATH=/opt/qt4.7/lib/ kvirc4
+  If you use KVIrc in a language other than English, then you need
+  the GNU gettext package. In particular KVIrc uses msgfmt.
 
-    #
-    # CMake
-    #
+  This is usually included in your distribution in Ubuntu or variants is libgettextpo-dev
+  and if your distro is not Ubuntu or variant you can check it by running:
 
-    This is usually included in nowadays distros.
-    To check which CMake you have, simply run:
+    $ msgfmt --version
 
-    # cmake --version
+  KVIrc will not complain if the command above is missing. It will just skip
+  the creation of the translation files. See the cmake configuration output.
+  If the command above fails, then you need to install the gettext package if
+  you want any other language than English.
```

Comment:
That information already was here, I just added "See cmake configuration output" Ill look at this again and improve it.